### DISCUSSION TITLE
Detekt Baseline Auto Clean-Up

### DISCUSSION THREAD

**Discussion by ghost** [detekt/detekt#5913] [General]
We created a detekt-baseline file. Sometimes as we improved new features, baselined issues had gone.
The team runs detekt task at every push. But baseline file won't reduce till run the detektBaseline task.
Is there a way to clean-up baseline automatically as we run detekt?

**Comment by cortinico** [comment]
> Is there a way to clean-up baseline automatically as we run detekt?

Not that I'm aware of 🤔 Yet this could be a nice feature to develop

**Comment by BraisGabin** [comment]
There isn't. On my project I have nighly that executes the detektBaseline task and commits any change.

**Comment by ghost** [comment]
@cortinico , @BraisGabin thanks for your response.
I would really appreciate that if you develop this feature. Reducing baseline file automatically would be great since we don't aware that we solved an issue that baselined

**Reply by cortinico** [reply]
We're happy to receive a contribution :)
